src/mqt/ddsim/target.py

```diff
@@ -0,0 +1,130 @@
+"""Utilities for constructing a DDSIM target."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import qiskit.circuit.library as qcl
+from qiskit.circuit import Parameter
+
+if TYPE_CHECKING:
+    from qiskit.transpiler import Target
+
+
+class DDSIMTargetBuilder:
+    @classmethod
+    def add_0q_gates(cls, target: Target) -> None:
+        target.add_instruction(qcl.GlobalPhaseGate(Parameter("phase")))
```

Comment:
Just thought about this a little more. Maybe instead of bumping the minimally supported Qiskit version, we can just add a check for the GlobalPhaseGate and only add it when it is available in the circuit library. Something like:
```suggestion
        try:
            target.add_instruction(qcl.GlobalPhaseGate(Parameter("phase")))
        except AttributeError:
            pass    
```
should make the minimums test pass.